Bottom: c069282288e753e9d38ef3c5352bdfb9efdfc3d0
Top:    ba906e2930dbb4d7f4fac11e14e1cda8e7f6da78
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 20:19:13 +0000

Remove debug from outline generation


---

diff --git a/src/polygon.c b/src/polygon.c
index 5e4e1fc..2ba4f52 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -108,7 +108,7 @@ dicer output is used for HIDs which cannot render things with holes
 #include <sys/time.h>
 #include <sys/resource.h>
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
